lib/components/agecounters.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+var HOUR = 3600;
+var DAY  = HOUR * 24;
+
+/**
+ * A small helper object that counts into buckets the number of calls to its
+ * <code>bump</code> that fall into three age categories; ages within an hour,
+ * within a day or within seven days. Counts are maintained within the object,
+ * and can be fetched to set into a gauge metric object.
+ *
+ * This class is useful when exporting metrics that count the number of
+ * hourly/daily/weekly active instances of various types of object within the
+ * bridge.
+ *
+ * @constructor
+ */
+function AgeCounters() {
+    this["1h"] = 0;
+    this["1d"] = 0;
+    this["7d"] = 0;
+    this["all"] = 0;
```

Comment:
What is the intention behind assigning counters like this as opposed to:
```js
function AgeCounters() {
  this.counters = {
    "1h": 0,
    "1d": 0,
    "7d": 0,
    all: 0,
  }
}
```
I care because in its current form, `:48` is rather error prone if we ever add **any** new property to `AgeCounters`. If we namespaced it all under `counters`, this would make `Object.keys(this)` much less error-prone because it would be replaced with `Object.keys(this.counters)`, meaning new properties wouldn't clash.